Fix ZMI Templates for Groups/Users/Roles

petschki opened the pull request.

petschki committed:
Fix ZMI Templates for Groups/Users/Roles

dataflake reviewed (changes requested):
Thank you for this work. I'm noticing a couple things:

- on both the ZODB Role Manager and the ZODB Group Manager, on the view to assign a user to a group/role there's multiple arrow icons between the "Available" and "Active" boxes - I have seen this happening if the `<i>` that has the class is closed like `<i ... />`. Try closing it like `<i...></i>` instead.

- in the ZODB Role Manager "Roles" tab, the button should be named "Remove Roles" instead of "Remove role assignment"

petschki committed:
more ZMI cleanup

petschki commented:
@dataflake thank you for the review. I've changed the wording of the button.

Multiple icons are strange. I could not reproduce that. Though I've changed the `<i>` tag closing as you suggested.

The new commit is mainly table class cleanup and I've remove the `here/manage_tabs` from every modal add form. But I think this could need a bit more cleanup, because the `Design Hint` is still there because of the missing `<main>` block ...

dataflake commented:
I may be able to help out a bit more today or tomorrow.

After encountering the double icons during similar work on Products.PluginRegistry I looked at the documentation and noticed they all use explicitly closed `</i>` in their examples. And in that case it made it work for me as well. Maybe a browser issue.

dataflake committed:
- more Zope 4 ZMI cleanups
